File: CQRSPerson.Domain/Entities/Person.cs
using System;
using System.Collections.Generic;

namespace CQRSPerson.Domain.Entities
{
    public partial class Person
    {
        public int PersonId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public string Interests { get; set; }
        public string Image { get; set; }
    }
}

File: CQRSPerson.UI/ClientApp/src/app/models/person.ts
export class Person{
    personId : number;
    firstName : string;
    lastName : string;
    age : string;
    interests : string;
    image : string;

    constructor(json : any){
        if(json){
            this.personId = json.personId;
            this.firstName = json.firstName;
            this.lastName = json.lastName;
            this.age = json.age;
            this.interests = json.interests;
            this.image = json.image;
        }
    }
}